scipy/stats/tests/test_stats.py

```diff
@@ -169,68 +169,93 @@ def test_tstd(self):
         y = stats.tstd(X, limits=None)
         assert_approx_equal(y, X.std(ddof=1), significant=self.dprec)
 
-    def test_tmin(self):
-        assert_equal(stats.tmin(4), 4)
```

Comment:
All of these were straightforward except the scalar and `nan_policy` checks got moved to separate tests.

These could easily be parametrized, but I'm keeping it simpler here.